### PR TITLE
Update datamarts-get-started.md

### PR DESCRIPTION
**Pull request by salilkanade**
Added "Adding or removing objects to the default dataset" and "Using model view layouts" sections

### CODE REVIEW

File: powerbi-docs/transform-model/datamarts/datamarts-get-started.md [hunk 1]
```diff
@@ -73,6 +73,30 @@ To model your data, navigate to **Model view** by selecting on the **Model View*
 
 :::image type="content" source="media/datamarts-get-started/datamarts-get-started-14.png" alt-text="Screenshot of the model view icon.":::
 
+### Adding or removing objects to the default dataset
+
+In Power BI, a dataset is always required before any reports can be built, so the default dataset enables quick reporting capabilities on top of the datamart.  Within the datamart, a user can add datamart objects -  tables to their default dataset. They can also add additional semantic modeling properties, such as hierarchies and descriptions.  These are then used to create the Power BI dataset’s tables. Users can also remove objects from the default dataset. 
+
+To add objects – tables or views to the default dataset, a user has 2 options:
+1.	Automatically add objects to the dataset, which happens by default with no user intervention needed
+2.	Manually add objects to the dataset
```
Comment (ShannonLeavitt) [inline powerbi-docs/transform-model/datamarts/datamarts-get-started.md:82]: Lines 81-82 must be a bulleted list.

**Comment by ShannonLeavitt** [comment]
@davidiseminger 

IMPORTANT: When the changes are ready for publication, add a `#sign-off` comment to signal that the PR is ready for the review team to merge. 

#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

**Comment by salilkanade** [comment]
Why is this not accepted? Are there any blockers still? @davidiseminger

**Comment by davidiseminger** [comment]
Looks good!
#sign-off

File: powerbi-docs/transform-model/datamarts/datamarts-get-started.md (hunk 1 above)
Comment (Jak-MS) [inline powerbi-docs/transform-model/datamarts/datamarts-get-started.md:82]:
```suggestion

- Automatically add objects to the dataset, which happens by default with no user intervention needed
- Manually add objects to the dataset

```

File: powerbi-docs/transform-model/datamarts/datamarts-get-started.md [hunk 2]
```diff
@@ -73,6 +73,30 @@ To model your data, navigate to **Model view** by selecting on the **Model View*
...
+
+The auto detect experience determines any tables or views and opportunistically adds them.
+
+The manually detect option in the ribbon allows fine grained control of which object(s) – tables and/or views, should be added to the default dataset:
+*	Select all
+*	Filter for tables or views
+*	Select specific objects
```
Comment (Jak-MS) [inline powerbi-docs/transform-model/datamarts/datamarts-get-started.md:89]:
```suggestion

1. *	Select all
2. *	Filter for tables or views
3. *	Select specific objects 

```

File: powerbi-docs/transform-model/datamarts/datamarts-get-started.md [hunk 3]
```diff
@@ -73,6 +73,30 @@ To model your data, navigate to **Model view** by selecting on the **Model View*
...
+
+The manually detect option in the ribbon allows fine grained control of which object(s) – tables and/or views, should be added to the default dataset:
+*	Select all
+*	Filter for tables or views
+*	Select specific objects 
+
+To remove objects, a user can use the manually select button in the ribbon and:
+*	Un-select all
+*	Filter for tables or views 
+*	Un-select specific objects
```
Comment (Jak-MS) [inline powerbi-docs/transform-model/datamarts/datamarts-get-started.md:94]:
```suggestion

1. Un-select all
2. Filter for tables or views 
3. Un-select specific objects 

```

**Pull request merged**